svm-feature-set/src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub struct SVMFeatureSet {
     pub formalize_loaded_transaction_data_size: bool,
     pub disable_zk_elgamal_proof_program: bool,
     pub reenable_zk_elgamal_proof_program: bool,
+    pub replace_spl_token_with_p_token: bool,
```

Comment:
Is this field needed? It looks like the migration is all outside the SVM, and doesn't actually affect VM behavior